templates/todo/projects/csharp-cosmos-sql/.repo/bicep/infra/main.bicep

```diff
@@ -154,6 +185,6 @@ output AZURE_KEY_VAULT_ENDPOINT string = keyVault.outputs.endpoint
 output AZURE_KEY_VAULT_NAME string = keyVault.outputs.name
 output AZURE_LOCATION string = location
 output AZURE_TENANT_ID string = tenant().tenantId
-output REACT_APP_API_BASE_URL string = api.outputs.SERVICE_API_URI
+output REACT_APP_API_BASE_URL string = useAPIM ? apimApi.outputs.SERVICE_API_URI : api.outputs.SERVICE_API_URI
```

Comment:
Output USE_APIM var so it is set in .env file.

Reply:
Like this?
`output USE_APIM string = useAPIM ? 'true' : 'false'`

Reply:
yes